ADRs/avoid nextjs async API utilities.md

```
@@ -0,0 +1,6 @@
+Next.JS now provides async cookies(), headers() etc. The problem is that you can for example set cookies by:
+(await cookies()).set(MY_COOKIE) and this cookies gonne best SET despite what request will you return! After settign up the cookie there might be a 500 error that you are returning, and cookie is gonna be attached!!!
+
+reasons for NextJS to use async: https://github.com/vercel/next.js/pull/68812
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)